lib/pf/util.pm

```diff
@@ -751,16 +751,18 @@ sub pf_run {
     my $caller = ( caller(1) )[3] || basename($0);
     $caller =~ s/^(pf::\w+|main):://;
 
+    my $loggable_command = $command;
+    $loggable_command =~ s/$options{log_strip}/*obfuscated-information*/g; 
```

Comment:
Change s/$options{log_strip}/*obfuscated-information*/g to s/\Q$options{log_strip}\E/*obfuscated-information*/g

So perl does not try to interpret  log_strip as a regex